### ISSUE TITLE
[IOS] Safari doesn't redirect after Google Sign In credentials entered

### ISSUE THREAD

**Issue by diego-caceres** [fullstackreact/react-native-oauth#118]
I have already followed all the steps to configure Google Sign In for Android and iOS. 
The problem is specifically with Google Sign In in iOS, and currently only tested this on the simulator. 

After I ask the manager to authorize the login, Safari opens correctly asking for Google Credentials. After I enter the credentials, or if I choose an already saved account, Safari tries to redirect to another google page that fails, and shows a temporary pop up that says:
"Safari cannot open the page because the address is invalid". 
This is closed after a few seconds and Safari remains in google's search page. 

I have already tried the url scheme alone in Safari and the app is opened correctly, so I suppose that that's correctly configured. I don't know exactly which url is it trying to open, but I suppose is the one where it tells the user which access is he giving the app.

Any help is appreciated! Thanks.

**Comment by dlykoits** [comment]
have you found the solution?

**Comment by diego-caceres** [comment]
No, I haven't. I ended up switching to another library in the end, react-native-google-signin and react-native-fbsdk for Facebook.

**Comment by marlonbrgomes** [comment]
+1

**Comment by aethiss** [comment]
Some update about that issue ?

**Comment by briank621** [comment]
Getting the same issue...

**Comment by seviltagiyeva** [comment]
same issue

**Comment by trickeyd** [comment]
Same here. Any update?

**Comment by trickeyd** [comment]
Okay, in case it helps anyone, I was missing this: 
![screen shot 2018-10-31 at 17 26 50](https://user-images.githubusercontent.com/1107295/47803263-61941c80-dd32-11e8-9d3f-8cb8313e5c2e.png)
